tests/queries/0_stateless/02006_test_positional_arguments.reference

```diff
@@ -111,3 +111,7 @@ select substr('aaaaaaaaaaaaaa', 8) as a  group by a;
 aaaaaaa
 select substr('aaaaaaaaaaaaaa', 8) as a  group by substr('aaaaaaaaaaaaaa', 8);
 aaaaaaa
+select b from (select 5 as a, 'Hello' as b order by a);
+Hello
+select b from (select 5 as a, 'Hello' as b group by a);
```

Comment:
we're going to test the same query twice, is it intentional?

Reply:
group by / order by

Reply:
sorry, my bad